packages/map-popup/src/index.tsx

```diff
@@ -113,6 +115,7 @@ export function MapPopup({ configCompanies, entity, getEntityName, setLocation,
 
   return (
     <S.MapOverlayPopup>
+      <FocusTrapWrapper closePopup={closePopup} id={`${entity.id}-popup`}>
```

Comment:
This addition is producing an undesired side effect. When the "Stop Viewer" button or the "From Here"/"To Here" button get focused, the focus is stuck on that button. I think that's because `FocusTrapWrapper` only moves the focus to siblings, and the focus has to stay within a containing element.

Not sure what the fix for this should be without fixing FocusTrapWrapper, so should we revert that for now?

Reply:
Actually, I think the issue was coming from double quotes inside the mock data ids: 
`id: '"hub_1580"',`
I wrote a line in map-popup to remove these for the purposes of the focus trap wrapper and the problem seems to be solved.... maybe it's not in the right spot? Let me know what you think

Reply:
Good find! I'm good with that.